servers/navigation_server_2d.h

```diff
@@ -272,7 +272,12 @@ class NavigationServer2D : public Object {
 	virtual void init() = 0;
 	virtual void sync() = 0;
 	virtual void finish() = 0;
-	virtual void free(RID p_object) = 0;
+	virtual void free_rid(RID p_rid) = 0;
+#ifndef DISABLE_DEPRECATED
+	[[deprecated("Use `free_rid()` instead.")]] void free(RID p_rid) {
+		free_rid(p_rid);
+	}
```

Comment:
The deprecated blocks should probably take up as little space as possible.
```suggestion
	[[deprecated("Use `free_rid()` instead.")]] void free(RID p_rid) { free_rid(p_rid); }
```